chapters/3-package-information.md

```diff
@@ -946,56 +946,41 @@ Example:
 
 **3.21.3** Cardinality: Optional (one or many)
 
-**3.21.4** Data Format: `<category> <type> <locator>`
+**3.21.4** Data Format: `<category> <URI>`
 
-where:
-
-`<category>` is “SECURITY” | “PACKAGE-MANAGER” | “OTHER”
-
-`<type>` is an [idstring] that is defined in Appendix.
+In Tag:value format, values must match the following [ABNF][rfc5234].
+`URI` is from pRFC 3986][rfc3986-s3].
+`space` is from [Appendi IV](appendix-IV-SPDX-license-expressions.md#overview).
```

Comment:
`space` is getting added in #37, but if there is interest in getting this PR in first, I can reroll to require a single U+0020 space for the time being.